WcaOnRails/app/models/competition.rb

```diff
@@ -427,6 +445,10 @@ def dangerously_close_to?(c)
     self.kilometers_to(c) <= NEARBY_DISTANCE_KM_DANGER && days_until.abs < NEARBY_DAYS_DANGER
   end
 
+  def results_uploaded?
```

Comment:
What was the decision about performance with this method? Are you still using it? Either way, can we create an issue to remember to add a database field for this instead?